Bottom: 0a8a15a889d1cdde9306331f60a4c04a496db497
Top:    c4002ed5f01ca2c0f265695d50e4e9b052e6984e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 20:19:13 +0000

Remove debug from outline generation


---

diff --git a/src/polygon.c b/src/polygon.c
index 65edb50..2bca1f3 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -108,7 +108,7 @@ dicer output is used for HIDs which cannot render things with holes
 #include <sys/time.h>
 #include <sys/resource.h>
 
-#define DEBUG_CIRCSEGS
+#undef DEBUG_CIRCSEGS
 
 #define ROUND(x) ((long)(((x) >= 0 ? (x) + 0.5  : (x) - 0.5)))
